Traducir.Api/Controllers/StringsController.cs

```diff
@@ -160,8 +163,10 @@ private static string FixWhitespaces(string suggestion, string original)
                 return BadRequest(SuggestionCreationResult.EmptySuggestion);
             }
 
+            var canReview = await _authorizationService.AuthorizeAsync(User, "CanReview");
```

Comment:
pet peeve: why don’t you call .Succeed here so that canReview is a boolean?

Reply:
I think we can open another issue for cleaning code on all repo.